src/components/layout/CompanyClaim/CompanyClaim.module.scss

```diff
@@ -33,23 +33,25 @@
 
       .cartIcon {
         background-color: $primary;
-        width: 55px;
-        height: 50px;
+        width: 70px;
+        height: 65px;
         border-radius: 3px;
         display: flex;
         align-items: center;
         justify-content: center;
         font-size: 20px;
 
         .icon {
-          margin-right: 5px;
+          margin-right: 18px;
+          width: 27px;
+          height: 25px;
         }
       }
 
       .cartCounter {
-        width: 28px;
-        height: 27px;
-        border-radius: 14px;
+        width: 45px;
```

Comment:
Nie ustawiaj sztywnej szerokości, bo ona jest zależna od ilości cyfr wyświetlanych wewnątrz. Przy aktualnym ustawieniu, jeśli chcemy wyświetlić 1 cyfrę (a w sklepie będzie to z pewnością najczęstszy przypadek) to mamy bardzo duże koło, zupełnie nie potrzebnie. Lepiej jest to osiągnąć za pomocą paddingu. Przy większych liczbach to nie musi być kółko.